app/src/main/java/uk/org/openseizuredetector/SdData.java

```diff
@@ -201,10 +217,23 @@ public boolean fromJSON(String jsonStr) {
                 mHRAlarmStanding = jo.optBoolean("hrAlarmStanding");
                 mHRThreshMax = jo.optDouble("hrThreshMax");
                 mHRThreshMin = jo.optDouble("hrThreshMin");
+                if (jo.has("adaptiveHrAlarmActive") &&
```

Comment:
Why are we checking specifically for some items in the JsonObject, but not others?